eslint config for nextjs

aymanesarrar opened the pull request:
The next configuration already handles setting default values for the parser, plugins and settings properties. 
more here : https://nextjs.org/docs/basic-features/eslint#migrating-existing-config

Closes #1321

mfix22 commented:
@aymanesarrar should we update the script to be `next lint` instead of `eslint .`? Are we then able to remove a few dependencies?

mfix22 commented on .eslintrc.js:
```diff
@@ -1,19 +1,11 @@
-  parserOptions: {
-    requireConfigFile: false,
-    babelOptions: {
-      presets: ['next/babel'],
```
Are we able to remove the various babel/eslint dependencies from our `package.json`?

aymanesarrar replied:
no i don't think we should remove it from our package.json

mfix22 commented:
@aymanesarrar `yarn lint` no longer passes, which is a major requirement to fix issue #1321, as specified in the issue body.

aymanesarrar commented:
i updated it to next lint and I had a problem that it doesn't know the extends next so I updated it to next/core-web-vitals but It requires to install eslint-next-config I've done it and npm run lint passes with some eslint errors, should i commit ?
